main.go

```diff
@@ -56,6 +57,9 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+//go:embed VERSION
+var version string
+
```

Comment:
Oh this is really nifty. Do you think we can/should converge this with the other way we get the version via a build flag?
https://github.com/ngrok/kubernetes-ingress-controller/blob/main/internal/version/version.go#L11
https://github.com/ngrok/kubernetes-ingress-controller/blob/main/.github/workflows/docker.yaml#L51

Reply:
👍 to consolidating either way. My preference would be on the internal version package so we can get the git sha as well

Reply:
Oops, my repo wasn't up to date! Explains why I didn't find this.

Moved the VERSION file to that package and added a symlink to the repo root, and replaced the ldflag with the embed.